app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,15 @@
+class WishlistsController < ApplicationController
+  def new
+    @wishlist = Wishlist.new
+  end
+
+  def create
+    @wishlist = Wishlist.new(wishlist_params)
+    @wishlist.save
```

Comment:
I'd redirect to a new `#new` view after success. Just for now, I'll update it when the `#index` view will be added.